cpp/tensorrt_llm/plugins/bertAttentionPlugin/bertAttentionPlugin.cpp

```diff
@@ -520,7 +520,7 @@ int BertAttentionPlugin::enqueueImpl(nvinfer1::PluginTensorDesc const* inputDesc
                 cudaMemsetAsync(fmhaParams.outputPtr, 0, ring_block_output_size, stream);
                 cudaMemcpyAsync(fmhaParams.tileCounterPtr, fmha_scheduler_counter_h, sizeof(uint32_t),
                     cudaMemcpyHostToDevice, stream);
-                mFMHARunner->run(fmhaParams);
+                mFmhaDispatcher->run(fmhaParams);
```

Comment:
Just to verify if I understand it correctly @PerkzZheng : the new FMHA dispatcher encapsulates the old FMHARunner & trtllmGenRunner, so it can run both fmha_v2 (<= sm90) and trtllm-gen (>=sm100)

Reply:
yes.